### PR TITLE
Sponsor Details Field Group now in wp sponsored post plugin

### PR DESCRIPTION
**Pull request by leonoragilmour**
Renaming so that diggers doesn't go mad...!

Also I've commented out the colour fields cause they're not yet approved I don't think??

Once this is accepted, will copy over to the other sponsored plugins...

### CODE REVIEW

**Comment by leonoragilmour** [comment]
[Update sponsor details in API](https://trello.com/c/xlOHRO70/232-update-sponsor-details-in-api)

File: app/CustomFields/SponsorDetails.php [hunk 1]
```diff
@@ -7,7 +7,7 @@ public function init() {
 
 		add_action('agreable_app_theme_init', function() {
 
-			$key = 'sponsored_longform_sponsor_details';
+			$key = 'sponsor_details';
```
Comment (ecoad) [inline app/CustomFields/SponsorDetails.php]: > am I right in thinking that these post types are not currently used on any sites? Will need to write a migration if they have!
that's right, this is a new ACF Group

> And is / does catfish import partnerships as sponsored longforms?
yes

Comment (leonoragilmour) [inline app/CustomFields/SponsorDetails.php]: ok so migrations are not needed but I will have to update catfish

**Comment by leonoragilmour** [comment]
This PR is now the main field group
https://github.com/shortlist-digital/wp-sponsored-post-plugin/pull/5

The most recent commits in this PR have removed the field group and instead used hooks to access the sponsored post plugin filter.

**Comment by thomasdigby** [comment]
Can we remove the social links as well? Or is this required by commercial? @WillBays @leonoragilmour

**Comment by WillBays** [comment]
Leave them out if it's easier for 4.0. We can pick them up post launch as necessary.

**Comment by leonoragilmour** [comment]
@WillBays @thomasdigby have now commented out social links https://github.com/shortlist-digital/wp-sponsored-post-plugin/pull/5

**Pull request merged**